src/main/java/com/google/api/codegen/csharp/CSharpGapicContext.java

```diff
@@ -596,7 +596,7 @@ private PageStreamerInfo getPageStreamerInfo(InterfaceConfig interfaceConfig, Me
     final InterfaceConfig interfaceConfig = getApiConfig().getInterfaceConfig(service);
     // TODO: Change back to .from(service.getMethods()) once streaming is implemented.
```

Comment:
No, not yet. It's saying to use `service.getMethods()` -- I think it is getting clearer since I've modified the method name to `getSupportedMethods` thanks to garrett's suggestion.